src/core/src/bootstrap/Constants.py

```diff
@@ -114,13 +114,12 @@ class ConfigSettings(EnumBackport):
     MAX_ASSESSMENT_RETRY_COUNT = 5
     MAX_INSTALLATION_RETRY_COUNT = 3
 
-    # Package Classifications
-    PACKAGE_CLASSIFICATIONS = {
-        0: 'Unclassified',           # doesn't serve a functional purpose in bit mask, but stands in for 'All' in code
-        1: 'Critical',
-        2: 'Security',
-        4: 'Other'
-    }
+    class PackageClassification(EnumBackport):
+        UNCLASSIFIED = 'Unclassified'
```

Comment:
I don't expect this is needed anymore (it had a specific reason to be in the older dictionary).

Reply:
It's used in a few tests, will have to modify those and ensure everything works fine. Will change it in the next PR.